CONTRIBUTING.md

```diff
@@ -46,16 +46,49 @@ git push fork feature
 
 Open a pull request against the main `opentelemetry-cpp-contrib` repo.
 
-### How to Receive Comments
+### Build and Run Code Examples
+
+TBD
+
+## Testing
+
+TBD
+
+## Contributing Rules
+
+Follow the OpenTelemetry [C++ SDK coding standards](https://github.com/open-telemetry/opentelemetry-cpp?tab=readme-ov-file#supported-c-versions)
+
+Include tests for new features or bug fixes
+
+
+## How to Receive Comments
 
 * If the PR is not ready for review, please put `[WIP]` in the title, tag it
   as `work-in-progress`, or mark it as [`draft`](https://github.blog/2019-02-14-introducing-draft-pull-requests/).
 * Make sure [CLA](https://identity.linuxfoundation.org/projects/cncf) is
   signed and CI is clear.
+  
+## How to Get PRs Merged
+
+Address any reviewer 
+
+Ensure all tests pass
+
+Squash and merge your commit when approved
+
+
+
+## Further Help
+
+The OpenTelemetry C/C++ special interest group (SIG) meets regularly. See the
+OpenTelemetry [community](https://github.com/open-telemetry/community#cc-sdk)
+repo for information on this and other language SIGs.
+
+See the [public meeting notes](https://docs.google.com/document/d/1i1E4-_y4uJ083lCutKGDhkpi3n4_e774SBLi9hPLocw/edit)
+for a summary description of past meetings. To request edit access, join the
+meeting or get in touch on [Gitter](https://gitter.im/open-telemetry/opentelemetry-cpp).
```

Comment:
We no longer use gitter.

Instead, we use the CNCF slack channel for opentelemetry-cpp